src/main/osd/osd_elements.c

```diff
@@ -1080,14 +1082,15 @@ static void osdElementGpsSpeed(osdElementParms_t *element)
 
 static void osdElementEfficiency(osdElementParms_t *element)
 {
-    int efficiency = 0;
+    int efficiency = 0.0;
```

Comment:
```suggestion
    int efficiency = 0.0f;
```

Reply:
Good catch.  Shouldn't this be:

int efficiency = 0;